src/java.desktop/windows/classes/sun/awt/Win32FontManager.java

```diff
@@ -47,24 +45,21 @@
  */
 public final class Win32FontManager extends SunFontManager {
 
-    @SuppressWarnings("removal")
-    private static final TrueTypeFont eudcFont =
-            AccessController.doPrivileged(new PrivilegedAction<TrueTypeFont>() {
-                public TrueTypeFont run() {
-                    String eudcFile = getEUDCFontFile();
-                    if (eudcFile != null) {
-                        try {
-                            /* Must use Java rasteriser since GDI doesn't
-                             * enumerate (allow direct use) of EUDC fonts.
-                             */
-                            return new TrueTypeFont(eudcFile, null, 0,
-                                                        true, false);
-                        } catch (FontFormatException e) {
-                        }
-                    }
-                    return null;
-                }
-            });
+    private static TrueTypeFont eudcFont;
```

Comment:
Minor: This can still be kept final by using a local variable (initialized to null) in the static block and assigning this field to that local variable once at the end of the block.

Alternatively, you can use a Supplier< TrueTypeFont()> lambda and make even fewer changes (it will be closer to the original). Something like this, which we did in a couple places in FX:

```
        private static final TrueTypeFont eudcFont =
            ((Supplier<TrueTypeFont>) () -> {
                String eudcFile = getEUDCFontFile();
                if (eudcFile != null) {
                    try {
                        /* Must use Java rasteriser since GDI doesn't
                         * enumerate (allow direct use) of EUDC fonts.
                         */
                        return new TrueTypeFont(eudcFile, null, 0,
                                                    true, false);
                    } catch (FontFormatException e) {
                    }
                }
                return null;
            }).get();
```

Reply:
ok..will be using this then..Thanks..